tests/server/metrics-errors.js

```diff
@@ -0,0 +1,48 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define([
+  'intern',
+  'intern!object',
+  'intern/chai!assert',
+  'intern/dojo/node!../../server/lib/configuration',
+  'intern/dojo/node!request'
+], function (intern, registerSuite, assert, config, request) {
+  var serverUrl = intern.config.fxaContentRoot.replace(/\/$/, '');
+
+  var metricsSampleRate = config.get('sentry.sample_rate');
```

Comment:
sentry.sample_rate is not actually defined in [configuration.js](https://github.com/mozilla/fxa-content-server/pull/2529/files#diff-a7802dc9db070ba75283c9fc3199eb02R80), this test throws an error:


> fxa-content-server.CRITICAL: Error: cannot find configuration param 'sentry.sample_rate'